lib/index.css

```diff
@@ -16,14 +16,149 @@
 
 @import url(lib/bike-rental.css);
 
+@import url(react-sliding-pane/dist/react-sliding-pane.css);
+
+
 /* Hide IE/Edge clear button in text input fields. */
 input[type=text]::-ms-clear {
     display: none;
 }
 
+/* New app menu */
+.app-menu-icon {
+  width: 21px;
+  height: 15px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  cursor: pointer;
+  position: absolute;
+  z-index: 10;
+  transition: all 1s ease;
+  top: 16px;
+}
```

Comment:
Tricky: insert `margin-left: 10px;` or `15px;` for the mobile view only (otherwise in mobile view there is no space to the left of the hamburger.)

Reply:
Good catch! Added in 542d756098ceae9ad56f74334fae242b4be7c938